### PR TITLE
Introduces a terse commit feed view

### PR DESCRIPTION
**Pull request by hennevogel**
Single line text instead of multi line HTML. Useful if you just want to know what's going on.

### CODE REVIEW

**Comment by hennevogel** [comment]
This is a bit too verbose for my taste (on slack)

Screenshot from 2021-09-06 17-45-26

File: src/api/app/controllers/webui/feeds_controller.rb [hunk 1]
```diff
@@ -15,12 +15,13 @@ def latest_updates
   def commits
     authorize @project, :source_access?
 
-    @start = params[:starting_at].present? ? starting_at(params[:starting_at]) : 7.days.ago
-    @finish = params[:ending_at].present? ? ending_at(params[:ending_at]) : nil
+    start = params[:starting_at].present? ? starting_at(params[:starting_at]) : 7.days.ago
```
Comment (vpereira) [inline src/api/app/controllers/webui/feeds_controller.rb:18]: maybe would be simple with a `starting_at(params.fetch(:starting_at, 7.days.ago.to_s))` ?

File: src/api/app/controllers/webui/feeds_controller.rb [hunk 2]
```diff
@@ -15,12 +15,13 @@ def latest_updates
...
+    finish = params[:ending_at].present? ? ending_at(params[:ending_at]) : nil
+    @terse = params[:terse].present?
 
-    @commits = @project.project_log_entries.where(event_type: 'commit').where(['datetime >= ?', @start])
-    @commits = @commits.where(['datetime <= ?', @finish]) unless @finish.nil?
-    @commits = @commits.order('datetime desc')
+    commits = @project.project_log_entries.where(event_type: 'commit').where(['datetime >= ?', start])
+    commits = commits.where(['datetime <= ?', finish]) unless finish.nil?
```
Comment (vpereira) [inline src/api/app/controllers/webui/feeds_controller.rb:23]: maybe simpler with `commits = commits.where(['datetime <= ?', finish]) if finish.present?`

File: src/api/app/controllers/webui/feeds_controller.rb (hunk 1 above)
Comment (hennevogel) [reply src/api/app/controllers/webui/feeds_controller.rb]: `before_action` to the rescue...

File: src/api/app/controllers/webui/feeds_controller.rb [hunk 3]
```diff
@@ -38,15 +38,14 @@ def notifications
...
-  end
-
-  def ending_at(date)
-    Time.zone.parse(date)
-  rescue StandardError
-    nil
+  def set_timerange
+    start = params.fetch(:starting_at, 7.days.ago.to_s)
+    @start = Time.zone.parse(start)
+    finish = params.fetch(:ending_at, nil)
```
Comment (dmarcoux) [inline src/api/app/controllers/webui/feeds_controller.rb:44]: Using `fetch` isn't needed if you set the default value to `nil`:

```suggestion
    finish = params[:ending_at]
```

Comment (hennevogel) [reply src/api/app/controllers/webui/feeds_controller.rb:44]: https://api.rubyonrails.org/classes/ActionController/Parameters.html#method-i-fetch

Comment (hennevogel) [reply src/api/app/controllers/webui/feeds_controller.rb]: but as you wish :)

**Review by vpereira — APPROVED** [review]

**Pull request merged**